integrations/github/src/sync.ts

```diff
@@ -43,10 +43,18 @@ export async function triggerImport(
 
         /** Whether the git info should be updated on the space */
         updateGitInfo?: boolean;
+
+        /**
+         * The date/time of the event that triggers the import.
+         *
+         * This is to help ensures that Git sync import and export operations are executed
+         * in the same order on GitBook and on the remote repository.
+         */
+        eventCreatedAt?: Date;
```

Comment:
I think that `eventTimestamp` would be a slightly better name 🙃

Reply:
Yep agree, renamed it, thanks.